xml/admin_nfsganesha.xml

```diff
@@ -389,6 +405,80 @@ EXPORT {
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="ganesha-getting-exports">
+   <title>Getting Exports Configuration</title>
+   <para>
+    To get the exports configuration details, follow these steps:
```

Comment:
What does it mean though "To get"? Are you downloading? Receiving? Creating? Exporting?

Reply:
my previous reply clears this